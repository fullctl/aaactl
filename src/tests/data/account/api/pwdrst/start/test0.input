{
  "client": "api_client_anon",
  "data": {
    "email": "test@localhost"
  }
}
