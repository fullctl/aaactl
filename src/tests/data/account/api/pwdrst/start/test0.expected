{
  "status": 200,
  "response": {"data": [{"email": "test@localhost"}], "errors": {}, "profiling": {"queries": 0}} 
}
